### ISSUE TITLE
route syntax validation should s/explain when it it fails 

### ISSUE THREAD

**Issue by bhurlow** [weavejester/ataraxy#1]
I've found it a bit hard to debug the routing syntax when it fails to pass the `valid?` pre condition: https://github.com/weavejester/ataraxy/blob/master/src/ataraxy/core.clj#L65-L68

**Comment by bhurlow** [comment]
I resorted to: `(s/explain :ataraxy.core/routing-table-with-meta myroutes)`  manually

**Comment by weavejester** [comment]
Yes, this is something I've been meaning to add, but the `explain` output gets a little messy around some of the rules, so I wanted to clean it up before making the validation a little better.

**Comment by bhurlow** [comment]
somebody needs to make a pretty parser for the spec output! it's almost ironic how hard to read the explain result it
